sbws/lib/v3bwfile.py

```diff
@@ -887,10 +895,13 @@ def rm_link(self, output):
             except FileNotFoundError:
                 pass
 
-    def write(self, output):
+    def write(self, output, rm_link=False):
         if output == '/dev/stdout':
             log.info("Writing to stdout is not supported.")
             return
+        if not self.is_min_perc and rm_link:
```

Comment:
Does sbws need to be able to remove the link?
It seems simpler and more reliable to always write the file, and always have the link.